tests/Doctrine/Tests/ORM/Tools/SchemaValidatorTest.php

```diff
@@ -200,6 +200,21 @@ public function testInvalidOrderByAssociationInverseSide()
             $ce
         );
     }
+
+    public function testInvalidReferencedJoinTableColumnIsNotPrimary() : void
+    {
+        $class = $this->em->getClassMetadata(InvalidEntity3::class);
+        $ce = $this->validator->validateClass($class);
```

Comment:
We could have a better name for this, maybe `$errors`? I know that it's the same that we have in the implementation, but that doesn't mean it's a good choice 😂

Reply:
Just a lazy copy&paste. Renamed all occurrences.